resources/util.sh

```diff
@@ -92,6 +92,18 @@ function configureNexusAtFirstStart() {
       doguctl template "${NEXUS_WORKDIR}/resources/nexusConfParameters.json.tpl" \
       "${NEXUS_WORKDIR}/resources/nexusConfParameters.json"
 
+    echo "Rendering cleanupPolicies template"
+    ADMINDEFAULTPASSWORD="${nexusPassword}" \
+      NEWADMINPASSWORD="${ADMINPW}" \
```

Comment:
Where will these variables be used? The templates itself don't need them.

Reply:
Yes but on first run the script needs (god knows why the credentials) I had tried it in between without and it didn't run then. But I can try it again.